<?xml version="1.0" encoding="UTF-8" ?>
<testsuite tests="5" failures="1" name="com.fullstack.project.dao.jdbc.impl.PersonDaoImplTest" time="0.049" errors="4" skipped="0">
  <properties>
    <property name="java.runtime.name" value="OpenJDK Runtime Environment"/>
    <property name="java.vm.version" value="18.0.2.1+1-1"/>
    <property name="sun.boot.library.path" value="/Users/SerenaPang/Library/Java/JavaVirtualMachines/openjdk-18.0.2.1/Contents/Home/lib"/>
    <property name="maven.multiModuleProjectDirectory" value="/Users/SerenaPang/Development/basic-fullstack-project-main"/>
    <property name="java.vm.vendor" value="Oracle Corporation"/>
    <property name="java.vendor.url" value="https://java.oracle.com/"/>
    <property name="guice.disable.misplaced.annotation.check" value="true"/>
    <property name="path.separator" value=":"/>
    <property name="java.vm.name" value="OpenJDK 64-Bit Server VM"/>
    <property name="user.country" value="CN"/>
    <property name="sun.java.launcher" value="SUN_STANDARD"/>
    <property name="java.vm.specification.name" value="Java Virtual Machine Specification"/>
    <property name="user.dir" value="/Users/SerenaPang/Development/basic-fullstack-project-main"/>
    <property name="java.vm.compressedOopsMode" value="Zero based"/>
    <property name="java.runtime.version" value="18.0.2.1+1-1"/>
    <property name="os.arch" value="x86_64"/>
    <property name="java.io.tmpdir" value="/var/folders/cc/kd7xwgrx29g7622p4krg5nxh0000gn/T/"/>
    <property name="line.separator" value="
"/>
    <property name="java.vm.specification.vendor" value="Oracle Corporation"/>
    <property name="os.name" value="Mac OS X"/>
    <property name="classworlds.conf" value="/Users/SerenaPang/Downloads/apache-maven-3.8.6/bin/m2.conf"/>
    <property name="sun.jnu.encoding" value="UTF-8"/>
    <property name="java.library.path" value="/Users/SerenaPang/Library/Java/Extensions:/Library/Java/Extensions:/Network/Library/Java/Extensions:/System/Library/Java/Extensions:/usr/lib/java:."/>
    <property name="maven.conf" value="/Users/SerenaPang/Downloads/apache-maven-3.8.6/conf"/>
    <property name="jdk.debug" value="release"/>
    <property name="java.class.version" value="62.0"/>
    <property name="java.specification.name" value="Java Platform API Specification"/>
    <property name="sun.management.compiler" value="HotSpot 64-Bit Tiered Compilers"/>
    <property name="os.version" value="12.4"/>
    <property name="library.jansi.path" value="/Users/SerenaPang/Downloads/apache-maven-3.8.6/lib/jansi-native"/>
    <property name="http.nonProxyHosts" value="local|*.local|169.254/16|*.169.254/16"/>
    <property name="user.home" value="/Users/SerenaPang"/>
    <property name="user.timezone" value="Asia/Shanghai"/>
    <property name="file.encoding" value="UTF-8"/>
    <property name="java.specification.version" value="18"/>
    <property name="user.name" value="pangyongxin"/>
    <property name="java.class.path" value="/Users/SerenaPang/Downloads/apache-maven-3.8.6/boot/plexus-classworlds-2.6.0.jar"/>
    <property name="java.vm.specification.version" value="18"/>
    <property name="sun.arch.data.model" value="64"/>
    <property name="sun.java.command" value="org.codehaus.plexus.classworlds.launcher.Launcher assembly:assembly -DdescriptorId=jar-with-dependencies"/>
    <property name="java.home" value="/Users/SerenaPang/Library/Java/JavaVirtualMachines/openjdk-18.0.2.1/Contents/Home"/>
    <property name="apple.awt.application.name" value="Launcher"/>
    <property name="user.language" value="en"/>
    <property name="java.specification.vendor" value="Oracle Corporation"/>
    <property name="java.vm.info" value="mixed mode, sharing"/>
    <property name="descriptorId" value="jar-with-dependencies"/>
    <property name="java.version" value="18.0.2.1"/>
    <property name="native.encoding" value="UTF-8"/>
    <property name="java.vendor" value="Oracle Corporation"/>
    <property name="sun.stderr.encoding" value="UTF-8"/>
    <property name="maven.home" value="/Users/SerenaPang/Downloads/apache-maven-3.8.6"/>
    <property name="file.separator" value="/"/>
    <property name="java.version.date" value="2022-08-18"/>
    <property name="java.vendor.url.bug" value="https://bugreport.java.com/bugreport/"/>
    <property name="sun.io.unicode.encoding" value="UnicodeBig"/>
    <property name="sun.cpu.endian" value="little"/>
    <property name="sun.stdout.encoding" value="UTF-8"/>
    <property name="socksNonProxyHosts" value="local|*.local|169.254/16|*.169.254/16"/>
    <property name="ftp.nonProxyHosts" value="local|*.local|169.254/16|*.169.254/16"/>
  </properties>
  <testcase classname="com.fullstack.project.dao.jdbc.impl.PersonDaoImplTest" name="delete" time="0.016">
    <error message="java.sql.SQLSyntaxErrorException: (conn=128) You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near &apos;person(lastname) value(&apos;Carlos&apos;)&apos; at line 1" type="com.fullstack.project.exceptions.Exceptions$DbException">com.fullstack.project.exceptions.Exceptions$DbException: java.sql.SQLSyntaxErrorException: (conn=128) You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near &apos;person(lastname) value(&apos;Carlos&apos;)&apos; at line 1
	at com.fullstack.project.dao.jdbc.impl.PersonDaoImpl.insert(PersonDaoImpl.java:42)
	at com.fullstack.project.dao.jdbc.impl.PersonDaoImplTest.delete(PersonDaoImplTest.java:109)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
	at java.base/java.lang.reflect.Method.invoke(Method.java:577)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:271)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:50)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:252)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:141)
	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:112)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
	at java.base/java.lang.reflect.Method.invoke(Method.java:577)
	at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
	at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
	at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
Caused by: java.sql.SQLSyntaxErrorException: (conn=128) You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near &apos;person(lastname) value(&apos;Carlos&apos;)&apos; at line 1
	at org.mariadb.jdbc.export.ExceptionFactory.createException(ExceptionFactory.java:280)
	at org.mariadb.jdbc.export.ExceptionFactory.create(ExceptionFactory.java:368)
	at org.mariadb.jdbc.message.ClientMessage.readPacket(ClientMessage.java:137)
	at org.mariadb.jdbc.client.impl.StandardClient.readPacket(StandardClient.java:833)
	at org.mariadb.jdbc.client.impl.StandardClient.readResults(StandardClient.java:772)
	at org.mariadb.jdbc.client.impl.StandardClient.readResponse(StandardClient.java:691)
	at org.mariadb.jdbc.client.impl.StandardClient.execute(StandardClient.java:634)
	at org.mariadb.jdbc.ClientPreparedStatement.executeInternal(ClientPreparedStatement.java:95)
	at org.mariadb.jdbc.ClientPreparedStatement.executeLargeUpdate(ClientPreparedStatement.java:334)
	at org.mariadb.jdbc.ClientPreparedStatement.executeUpdate(ClientPreparedStatement.java:311)
	at com.fullstack.project.dao.jdbc.impl.PersonDaoImpl.insert(PersonDaoImpl.java:30)
	... 26 more
</error>
    <system-err>[ WARN] (main) Error: 1064-42000: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near &apos;person(lastname) value(&apos;Carlos&apos;)&apos; at line 1
</system-err>
  </testcase>
  <testcase classname="com.fullstack.project.dao.jdbc.impl.PersonDaoImplTest" name="insert" time="0.01">
    <error message="java.sql.SQLSyntaxErrorException: (conn=129) You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near &apos;person(lastname) value(&apos;Pang&apos;)&apos; at line 1" type="com.fullstack.project.exceptions.Exceptions$DbException">com.fullstack.project.exceptions.Exceptions$DbException: java.sql.SQLSyntaxErrorException: (conn=129) You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near &apos;person(lastname) value(&apos;Pang&apos;)&apos; at line 1
	at com.fullstack.project.dao.jdbc.impl.PersonDaoImpl.insert(PersonDaoImpl.java:42)
	at com.fullstack.project.dao.jdbc.impl.PersonDaoImplTest.insert(PersonDaoImplTest.java:34)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
	at java.base/java.lang.reflect.Method.invoke(Method.java:577)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:271)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:50)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:252)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:141)
	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:112)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
	at java.base/java.lang.reflect.Method.invoke(Method.java:577)
	at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
	at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
	at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
Caused by: java.sql.SQLSyntaxErrorException: (conn=129) You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near &apos;person(lastname) value(&apos;Pang&apos;)&apos; at line 1
	at org.mariadb.jdbc.export.ExceptionFactory.createException(ExceptionFactory.java:280)
	at org.mariadb.jdbc.export.ExceptionFactory.create(ExceptionFactory.java:368)
	at org.mariadb.jdbc.message.ClientMessage.readPacket(ClientMessage.java:137)
	at org.mariadb.jdbc.client.impl.StandardClient.readPacket(StandardClient.java:833)
	at org.mariadb.jdbc.client.impl.StandardClient.readResults(StandardClient.java:772)
	at org.mariadb.jdbc.client.impl.StandardClient.readResponse(StandardClient.java:691)
	at org.mariadb.jdbc.client.impl.StandardClient.execute(StandardClient.java:634)
	at org.mariadb.jdbc.ClientPreparedStatement.executeInternal(ClientPreparedStatement.java:95)
	at org.mariadb.jdbc.ClientPreparedStatement.executeLargeUpdate(ClientPreparedStatement.java:334)
	at org.mariadb.jdbc.ClientPreparedStatement.executeUpdate(ClientPreparedStatement.java:311)
	at com.fullstack.project.dao.jdbc.impl.PersonDaoImpl.insert(PersonDaoImpl.java:30)
	... 26 more
</error>
    <system-err>[ WARN] (main) Error: 1064-42000: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near &apos;person(lastname) value(&apos;Pang&apos;)&apos; at line 1
</system-err>
  </testcase>
  <testcase classname="com.fullstack.project.dao.jdbc.impl.PersonDaoImplTest" name="findAll" time="0.007">
    <failure type="java.lang.AssertionError">java.lang.AssertionError
	at org.junit.Assert.fail(Assert.java:86)
	at org.junit.Assert.assertTrue(Assert.java:41)
	at org.junit.Assert.assertTrue(Assert.java:52)
	at com.fullstack.project.dao.jdbc.impl.PersonDaoImplTest.findAll(PersonDaoImplTest.java:71)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
	at java.base/java.lang.reflect.Method.invoke(Method.java:577)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:271)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:50)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:252)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:141)
	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:112)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
	at java.base/java.lang.reflect.Method.invoke(Method.java:577)
	at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
	at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
	at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
</failure>
  </testcase>
  <testcase classname="com.fullstack.project.dao.jdbc.impl.PersonDaoImplTest" name="update" time="0.008">
    <error message="java.sql.SQLSyntaxErrorException: (conn=131) You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near &apos;person(lastname) value(&apos;P&apos;)&apos; at line 1" type="com.fullstack.project.exceptions.Exceptions$DbException">com.fullstack.project.exceptions.Exceptions$DbException: java.sql.SQLSyntaxErrorException: (conn=131) You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near &apos;person(lastname) value(&apos;P&apos;)&apos; at line 1
	at com.fullstack.project.dao.jdbc.impl.PersonDaoImpl.insert(PersonDaoImpl.java:42)
	at com.fullstack.project.dao.jdbc.impl.PersonDaoImplTest.update(PersonDaoImplTest.java:84)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
	at java.base/java.lang.reflect.Method.invoke(Method.java:577)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:271)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:50)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:252)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:141)
	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:112)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
	at java.base/java.lang.reflect.Method.invoke(Method.java:577)
	at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
	at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
	at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
Caused by: java.sql.SQLSyntaxErrorException: (conn=131) You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near &apos;person(lastname) value(&apos;P&apos;)&apos; at line 1
	at org.mariadb.jdbc.export.ExceptionFactory.createException(ExceptionFactory.java:280)
	at org.mariadb.jdbc.export.ExceptionFactory.create(ExceptionFactory.java:368)
	at org.mariadb.jdbc.message.ClientMessage.readPacket(ClientMessage.java:137)
	at org.mariadb.jdbc.client.impl.StandardClient.readPacket(StandardClient.java:833)
	at org.mariadb.jdbc.client.impl.StandardClient.readResults(StandardClient.java:772)
	at org.mariadb.jdbc.client.impl.StandardClient.readResponse(StandardClient.java:691)
	at org.mariadb.jdbc.client.impl.StandardClient.execute(StandardClient.java:634)
	at org.mariadb.jdbc.ClientPreparedStatement.executeInternal(ClientPreparedStatement.java:95)
	at org.mariadb.jdbc.ClientPreparedStatement.executeLargeUpdate(ClientPreparedStatement.java:334)
	at org.mariadb.jdbc.ClientPreparedStatement.executeUpdate(ClientPreparedStatement.java:311)
	at com.fullstack.project.dao.jdbc.impl.PersonDaoImpl.insert(PersonDaoImpl.java:30)
	... 26 more
</error>
    <system-err>[ WARN] (main) Error: 1064-42000: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near &apos;person(lastname) value(&apos;P&apos;)&apos; at line 1
</system-err>
  </testcase>
  <testcase classname="com.fullstack.project.dao.jdbc.impl.PersonDaoImplTest" name="findById" time="0.008">
    <error message="java.sql.SQLSyntaxErrorException: (conn=132) You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near &apos;person(lastname) value(&apos;Pang&apos;)&apos; at line 1" type="com.fullstack.project.exceptions.Exceptions$DbException">com.fullstack.project.exceptions.Exceptions$DbException: java.sql.SQLSyntaxErrorException: (conn=132) You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near &apos;person(lastname) value(&apos;Pang&apos;)&apos; at line 1
	at com.fullstack.project.dao.jdbc.impl.PersonDaoImpl.insert(PersonDaoImpl.java:42)
	at com.fullstack.project.dao.jdbc.impl.PersonDaoImplTest.findById(PersonDaoImplTest.java:49)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
	at java.base/java.lang.reflect.Method.invoke(Method.java:577)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:271)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:50)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:252)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:141)
	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:112)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
	at java.base/java.lang.reflect.Method.invoke(Method.java:577)
	at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
	at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
	at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
Caused by: java.sql.SQLSyntaxErrorException: (conn=132) You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near &apos;person(lastname) value(&apos;Pang&apos;)&apos; at line 1
	at org.mariadb.jdbc.export.ExceptionFactory.createException(ExceptionFactory.java:280)
	at org.mariadb.jdbc.export.ExceptionFactory.create(ExceptionFactory.java:368)
	at org.mariadb.jdbc.message.ClientMessage.readPacket(ClientMessage.java:137)
	at org.mariadb.jdbc.client.impl.StandardClient.readPacket(StandardClient.java:833)
	at org.mariadb.jdbc.client.impl.StandardClient.readResults(StandardClient.java:772)
	at org.mariadb.jdbc.client.impl.StandardClient.readResponse(StandardClient.java:691)
	at org.mariadb.jdbc.client.impl.StandardClient.execute(StandardClient.java:634)
	at org.mariadb.jdbc.ClientPreparedStatement.executeInternal(ClientPreparedStatement.java:95)
	at org.mariadb.jdbc.ClientPreparedStatement.executeLargeUpdate(ClientPreparedStatement.java:334)
	at org.mariadb.jdbc.ClientPreparedStatement.executeUpdate(ClientPreparedStatement.java:311)
	at com.fullstack.project.dao.jdbc.impl.PersonDaoImpl.insert(PersonDaoImpl.java:30)
	... 26 more
</error>
    <system-err>[ WARN] (main) Error: 1064-42000: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near &apos;person(lastname) value(&apos;Pang&apos;)&apos; at line 1
</system-err>
  </testcase>
</testsuite>